### DISCUSSION TITLE
Pass document into chat-cli chat mode

### DISCUSSION THREAD

**Discussion by WPrintz** [chat-cli/chat-cli#42] [Ideas]
It would be helpful to be able to pass a text document into chat-cli chat mode in order to converse about the contents of the file.  One example would be to invoke chat-cli like `chat-cli chat < meeting_transcript.txt` which would then provide ability to hold a dialog about the contents of the meeting transcript file, such as asking for key talking points, action items, etc.

**Comment by micahwalter** [comment]
This seems like an easy one. I'll do this!
